chapters/en/chapter5/fine-tuning_parakeet.mdx

```diff
@@ -0,0 +1,361 @@
+# Fine-tuning Parakeet for ASR
+
+In this section, we will explore how to fine-tune NVIDIA's Parakeet CTC model for Automatic Speech Recognition (ASR) on a new language. Parakeet is a family of high-performance ASR models based on the FastConformer architecture, trained on over 64,000 hours of English speech data. We will fine-tune it on Spanish speech data to demonstrate how you can adapt this powerful English model to recognize speech in other languages.
+
+## House-keeping
+
+To fine-tune a model of this size (1.1B parameters), a GPU is essential. You can check your GPU status with:
+
+```bash
+nvidia-smi
+```
+
+You will need to install a few specific dependencies:
+
+```bash
+pip install transformers datasets librosa jiwer peft accelerate soundfile
+```
+
+Make sure to log in to your Hugging Face account if you want to push your model to the Hub:
+
+```python
+from huggingface_hub import notebook_login
+
+notebook_login()
+```
+
+## The Dataset
+
+For this example, we will use the Spanish subset of the [Multilingual LibriSpeech (MLS)](https://huggingface.co/datasets/facebook/multilingual_librispeech) dataset. MLS is a large-scale multilingual corpus derived from LibriVox audiobooks, containing thousands of hours of transcribed speech in 8 languages.
+
+```python
+from datasets import load_dataset, Audio
+
+mls_train = load_dataset(
+    "facebook/multilingual_librispeech", "spanish", split="train", streaming=True
+)
+seed, buffer_size = 42, 10_000
+mls_train = mls_train.shuffle(seed=seed, buffer_size=buffer_size)
+mls_test = load_dataset(
+    "facebook/multilingual_librispeech", "spanish", split="test", streaming=True
+)
+```
+
+<Tip>
+    MLS supports 8 languages: German, Dutch, French, Spanish, Italian, Portuguese, Polish, and English.
+    Check the dataset card for more details: https://huggingface.co/datasets/facebook/multilingual_librispeech
+</Tip>
+
+We use streaming mode since MLS is a large dataset. Let's cast the audio to the correct sampling rate:
```

Comment:
"We use [streaming mode](https://huggingface.co/blog/streaming-datasets) (`streaming=True`) since MLS is a large dataset.

We will also cast the audio to the desired sampling rate:"